packages/cli/__tests__/zosfiles/__system__/create/ds/__snapshots__/cli.files.create.dataset.classic.system.test.ts.snap

```diff
@@ -0,0 +1,29 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Please delete this file since there isn't a system test anymore that matches its name.